src/components/TopBar/TopBarComponent.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import UploadIcon from 'components/TopBar/images/icon-uploads.svg';
+import SearchIcon from 'components/TopBar/images/icon-search.svg';
+import MenuIcon from 'components/TopBar/images/icon-menu.svg';
+import {
+  StyledControls,
+  StyledIcon,
+  StyledHeader,
+  StyledTopBar,
+  StyledTopBarWrapper,
+  StyledGradientLine,
+} from 'components/TopBar/TopBarStyles';
+
+export class TopBarComponent extends Component {
+  render() {
+    return (
+      <StyledTopBarWrapper>
+        <StyledTopBar>
+          <StyledHeader>NeuralNotes</StyledHeader>
+          <StyledControls>
+            <StyledIcon src={UploadIcon} />
+            <StyledIcon src={SearchIcon} />
+            <StyledIcon src={MenuIcon} width="12px" height="20px" />
```

Comment:
Why does this icon have different dimensions than the rest?

Reply:
It's not obvious, but, please look at these three icons
![image](https://user-images.githubusercontent.com/19979138/82836846-158a6900-9ed0-11ea-920d-5119ac86613e.png)
here -> [template](https://www.figma.com/file/NUWWByWR7Pfpq3h0v9oTMM/Neural-Notes-Design-Beta-1.0?node-id=218%3A83)
They have the same size of outer-box but the distance between this boxs is different. Since I'm using flexbox, I can edit the size of the elements but not their "between distance"

Reply:
Can we make this icon to be the same size as others? :)
I think the code will look nicer this way. More symmetric :)

Could you try editing `svg`'s dimensions?
![image](https://user-images.githubusercontent.com/6686679/82941033-5c469480-9f9e-11ea-9e3e-90fed0be90a5.png)

Reply:
I've already try it. It's not solve the problem. In this case we will get next view:
![image](https://user-images.githubusercontent.com/19979138/82949745-cc5c1700-9fac-11ea-8e5c-930e0dd7389c.png)

Reply:
What did you change? :)
What's wrong with how it looks like?

Reply:
We can merge it as is. Not a blocker.

Reply:
Ok, as you wish. But it still looks not perfect. I am killed by the different distances between the icons. Perhaps, as you are killed by the not perfect code)